modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/tests/BaseDmlDataTypeTest.java

```diff
@@ -89,19 +87,6 @@ public void testUpdateFromDynamicParam() {
                 .check();
     }
 
-    /**
-     * Type mismatch in {@code INSERT}s {@code VALUES} with dynamic parameters.
-     */
-    @ParameterizedTest
```

Comment:
Why did you decide to remove the test?

Reply:
mostly because configuration of these test cases is overcomplicated. Besides, there is no need to make it special only for certain types. Exhaustive validation test coverage is made by `DifferentFamiliesInsertSourcesCoercionTest` (I've added validation of multi-tuple insert, see changes in `DifferentFamiliesInsertSourcesCoercionTest`)